### PR TITLE
provide an actual code and configuration examples for split database connections

### PR DESCRIPTION
**Pull request by alfredodeza**
This was very tricky to get right and now that we are running it in production we thought it was going to be useful to share a good working example in addition to the current existing section about split databases

### CODE REVIEW

File: docs/source/databases.rst [hunk 1]
```diff
@@ -189,3 +191,85 @@ writing, and would always bind to a master database with read/write
...
+    }
+
+    # Pecan Application Configurations
+    app = {
+        ...
+    }
+
+    # Master database
+    sqlalchemy_w = {
+        'url': 'postgresql+psycopg2://root:@10.0.0.1/dbname',
```
Comment (ryanpetrello) [inline docs/source/databases.rst]: Maybe replace 10.0.0.x addresses here with example hostnames, e.g.,

postgresql+psycopg2://root:@master.dbhost/dbname

File: docs/source/databases.rst [hunk 2]
```diff
@@ -189,3 +191,85 @@ writing, and would always bind to a master database with read/write
...
+        'echo':          False,
```
Comment (ryanpetrello) [inline docs/source/databases.rst]: I'd also remove the echo and echo_pool here - pretty sure these are the default options in SQLAlchemy anyways

Comment (alfredodeza) [inline docs/source/databases.rst]: The examples at the beginning where using it, so I was trying to keep things consistent. But sure, I will update this.

File: docs/source/databases.rst [hunk 3]
```diff
@@ -189,3 +191,81 @@ writing, and would always bind to a master database with read/write
...
+    # Master database
+    sqlalchemy_w = {
+        'url': 'postgresql+psycopg2://root:@master_host/dbname',
+        'pool_recycle':  3600,
+        'encoding':      'utf-8'
+    }
+
+    # Read Only database
+    sqlalchemy_ro = {
+        'url': 'postgresql+psycopg2://root:@standy_host/dbname',
```
Comment (ryanpetrello) [inline docs/source/databases.rst]: s/standy/standby

Comment (alfredodeza) [inline docs/source/databases.rst]: I like `standy` it is like a way of standing that is hip

File: docs/source/databases.rst [hunk 4]
```diff
@@ -189,3 +191,81 @@ writing, and would always bind to a master database with read/write
...
+
+    # Pecan Application Configurations
+    app = {
+        ...
+    }
+
+    # Master database
+    sqlalchemy_w = {
+        'url': 'postgresql+psycopg2://root:@master_host/dbname',
+        'pool_recycle':  3600,
```
Comment (ryanpetrello) [inline docs/source/databases.rst]: This is kind of nitpick, but does the whitespace here (after the colon) get rendered in sphinx's output?

**Pull request merged**